src/stateManager/StateManager.ts

```diff
@@ -94,6 +94,7 @@ class StateManager {
     fraudProofService: FraudProofService;
     latestForkId: ForkId = NULL;
     defaultValidationStrategy: AValidationStrategy;
+    reductionTriggerMap: Map<ForkId, ReductionTimeoutHandle> = new Map();
```

Comment:
Maybe move this to storage

Reply:
in browsers it could be in storage since the handle for `setTimeout` is an number id - in node it's a reference so we'd lose that in storage + it's only used in the StateManager